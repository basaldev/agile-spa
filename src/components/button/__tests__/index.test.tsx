import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { Button } from '../index';

describe('Button', () => {
  test('render Button', () => {
      const wrapper = shallow(
        <Button label="label" onClick={jest.fn()} />
      );
      expect(toJson(wrapper)).toMatchSnapshot();
  });
});
