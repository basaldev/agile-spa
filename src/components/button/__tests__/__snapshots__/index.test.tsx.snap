// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button render Button 1`] = `
<styled.button
  onClick={[MockFunction]}
>
  label
</styled.button>
`;
